pages/blog.js

```diff
@@ -29,6 +29,7 @@ const blog = () => (
       </Grid>
     </ContentSection>
     <BlogList>
+      <BlogListItem slug="WI-2020-call-for-counselors-open" />
```

Comment:
Are we able to pull this from frontmatter so it isn't a static add?

Reply:
It's the name of the file, we could programmatically gather these but it's low of the to do list and pain point is super low.